src/main/java/seedu/duke/userinterface/command/List.java

```diff
@@ -1,8 +1,63 @@
 package seedu.duke.userinterface.command;
 
+import seedu.duke.notebooks.Notebook;
+import seedu.duke.notebooks.NotebookShelf;
+import seedu.duke.notebooks.Page;
+import seedu.duke.notebooks.Section;
+
 public class List extends CliCommand {
+
     @Override
     public void execute() {
 
```

Comment:
I think execute() should call the methods below as necessary, shouldn't it?

Reply:
This part I'm not sure. Could you explain how are we going to invoke the methods in this class?

Reply:
So when you want to list the notebooks, the CliUserInterface can do something like: 

```java
List listCommand = new List(commandArgs);
listCommand.execute();
```
And then in the execute method:
```java
public void execute() {
    switch(commandArguments) {
    case "nsp":
        listNotebooks_nsp();
        break;
    ...
    }
}
```

Not particularly good code up here, but I think you get the idea of what I meant, right?

Reply:
But the list() function needs the object as well, like listNotebooks_nsp(Notebook sampleNotebook), so I'm not sure how to code it. Also, should the List class handle input arguments like "nsp"? That doesn't seem follow the principle right?